playbooks/service/owncloud.yml

```diff
@@ -10,51 +10,69 @@
 
   roles:
 
+    - role: debops.php/env
+      tags: [ 'role::php', 'role::apt_preferences', 'role::logrotate' ]
+
+    - role: debops.owncloud/env
+      tags: [ 'role::owncloud', 'role::nginx' ]
+
+    - role: debops.apt_preferences
+      tags: [ 'role::apt_preferences' ]
+      apt_preferences__dependent_list:
+        - '{{ nginx_apt_preferences_dependent_list }}'
+        - '{{ owncloud__apt_preferences__dependent_list }}'
+        - '{{ php__apt_preferences__dependent_list }}'
+
+    - role: debops.ferm
+      tags: [ 'role::ferm' ]
+      ferm__dependent_rules:
+        - '{{ nginx_ferm_dependent_rules }}'
```

Comment:
`debops.nginx` now provides new variable for this, `nginx__ferm__dependent_rules`.